src/Runner/Runner.php

```diff
@@ -182,6 +183,8 @@ public function fix(): array
      */
     private function fixParallel(): array
     {
+        $this->dispatchEvent(AnalysisStarted::NAME, new AnalysisStarted(AnalysisStarted::MODE_PARALLEL, $this->isDryRun));
```

Comment:
If this PR represents `refactor` only, then `AnalysisStarted` and its dispatching shouldn't be part of it. But we can name the PR slightly different and then it will fit nicely anyway 🙂. Like "Enhance runner eventing system" or something like that.